.github/workflows/ci.yml

```
@@ -230,6 +230,11 @@ jobs:
         custom_script: |
           sudo apt-get update
           sudo apt-get install -y ghc cabal-install
+          
+          # See https://github.com/nodejs/help/issues/3644
+          curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash -
+          sudo apt-get install -y nodejs
+          
           sudo apt-get install -y --allow-unauthenticated \
           libnode-dev node-gyp libssl-dev npm aspcud      \
```

Do things break if we remove libnode-dev?
```suggestion
          node-gyp libssl-dev npm aspcud      \
```